src/platforms/javascript/guides/astro/manual-setup.mdx

```diff
@@ -0,0 +1,254 @@
+  dsn: "___PUBLIC_DSN___",
+  replaysSessionSampleRate: 0.1,
+  replaysOnErrorSampleRate: 1.0,
+  integrations: [
+    new Sentry.Replay({
+      maskAllText: true,
+      blockAllMedia: true,
+    }),
+    new BrowserTracing({
+      tracePropagationTargets: [/\//, "my-api-domain.com"],
+    }),
+  ],
+  tracesSampler: (samplingContext) => {
+    if (samplingContext.transactionContext.name === "/home") {
+      return 0.5;
+    }
+    return 0.1;
+  },
+});
+```
+
+### Server Side
+
+Here's an example of a custom server-side SDK setup:
+
+```javascript {filename:sentry.server.config.js}
+import * as sentry from "@sentry/astro";
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  tracesSampler: (samplingContext) => {
+    if (samplingContext.transactionContext.name === "/home") {
+      return 0.5;
+    }
+    return 0.1;
+  },
+  beforeSend: (event) => {
+    console.log("before send", event);
+    return event;
+  },
+});
+```
+
+### Changing config files location
+
+If you want to change the location of your `sentry.(client|server).config.js` files, specify the path to the files in the Sentry Astro integration options in your `astro.config.mjs`:
+
+```javascript {filename:astro.config.mjs}
+export default defineConfig({
+  // Rest of your Astro project config
+  integrations: [
+    sentryAstro({
+      clientInitPath: ".config/sentryClientInit.js",
+      serverInitPath: ".config/sentryServerInit.js",
+    }),
+  ],
+});
+```
+
+## Configure Source Maps Upload
+
+You can specify a few options to configure source maps upload for your production builds. Generally, this should work if you followed the [Astro CLI installation guide](../#add-readable-stack-traces-to-errors).
+
+Note that source maps upload only works if an auth token is specified.
+
+### Working with old authentication tokens
+
+If you're not using our new [organization-scoped auth tokens](/product/accounts/auth-tokens/#organization-auth-tokens) yet, you need to additionally specify your `org` slug in your `sourceMapsUploadOptions`:
+
+```javascript {filename:astro.config.mjs}
+export default defineConfig({
+  integrations: [
+    sentryAstro({
+      // rest of your sentry options
+      sourceMapsUploadOptions: {
+        project: "___PROJECT_SLUG___",
+        org: "___ORG_SLUG___",
+        authToken: process.env.SENTRY_AUTH_TOKEN,
+      },
+    }),
+  ],
+});
+```
+
+### Disable Source Maps Upload
+
+You can disable automatic source maps upload in your Vite config:
+
+```javascript {filename:astro.config.mjs}
+export default defineConfig({
+  integrations: [
+    sentryAstro({
+      // rest of your sentry options
+      sourceMapsUploadOptions: {
+        enabled: false,
+      },
+    }),
+  ],
+});
+```
+
+### Disabeling Telemetry Data Collection
+
+By default, our Vite plugin collects telemetry data to help us improve the source map uploading experience.
+Read more about this in our [Vite plugin documentation](https://www.npmjs.com/package/@sentry/vite-plugin#telemetry).
```

Comment:
Is this a typo? Or do we want this link because the Vite plugin is used under the hood? If so, let's note that here.

```suggestion
Read more about this in our [Astro plugin documentation](https://www.npmjs.com/package/@sentry/astro-plugin#telemetry).
```

Reply:
```suggestion
The Astro SDK uses the Sentry Vite Plugin to upload source maps. Read more about this in our [Vite plugin documentation](https://www.npmjs.com/package/@sentry/vite-plugin#telemetry).
```

Reply:
Oh I see, this goes on. I think I'll stick with Vite plugin but add a sentence on top that the Sentry Astro integration adds a Vite plugin under the hood for source maps upload.